interop/go.sum

```diff
@@ -2,10 +2,18 @@ cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMT
 github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03qcyfWMU=
 github.com/Hyperledger-TWGC/ccs-gm v0.0.0-20200917142018-eeebcfc2577c h1:E9Fce5+w9aNdHzFAy+rFKwW+t0f4idMj3zJb/JXRsfk=
```

Comment:
Can these be cleaned up? Will `go mod tidy` work?

Reply:
will try

Reply:
fixed.